M2/Macaulay2/d/actors4.d

```diff
@@ -663,10 +663,13 @@ stringcatfun(e:Expr):Expr := (
 setupfun("concatenate",stringcatfun);
 
 errorfun(e:Expr):Expr := (
-     e = stringcatfun(e);
      when e
      is s:stringCell do buildErrorPacket(s.v)
-     else buildErrorPacket("expects a string or sequence of strings as its argument"));
+     is err:Error do (
+	 err.printed = false;
+	 e)
```

Comment:
Sorry to be nitpicky but the indentation looks strange.
```suggestion
     is err:Error do ( err.printed = false; e )
```

Reply:
Fair enough.  I generally like to stick with one statement per line but `e` by itself is a little odd.  :) Fixed